### ISSUE TITLE
Invalid bundle generated with module augmentation and path aliases

### ISSUE THREAD

**Issue by DerGernTod** [timocov/dts-bundle-generator#209]
**Bug report**

**Input code**

Please see this reproducer repo (requires multiple modules): https://github.com/DerGernTod/dts-bundle-generator-aliases

**Expected output**

```ts
export const enum Foo {
	BAZ = "baz"
}
export declare const enum Foo {
	BAR = "bar"
}
export interface Augmented {
	[Foo.BAZ]: string;
}
export interface Augmented {
	[Foo.BAR]: string;
}

export {};
```

**Actual output**

```ts
declare module "@shared/foo" {
	const enum Foo {
		BAZ = "baz"
	}
}
declare module "@shared/foo-map" {
	interface Augmented {
		[Foo.BAZ]: string;
	}
}
export declare const enum Foo {
	BAR = "bar"
}
export interface Augmented {
	[Foo.BAR]: string;
}

export {};

```

**Additional context**
Running it with this config:
```
            noCheck: false,
            output: {
                inlineDeclareGlobals: true,
                sortNodes: true,
                noBanner: true,
                inlineDeclareExternals: true
            }
```
results in the error
```
dist/type-bundle.d.ts(8,3): error TS1169: A computed property name in an interface must refer to an expression whose type is a literal type or a 'unique symbol' type.
dist/type-bundle.d.ts(8,8): error TS2339: Property 'BAZ' does not exist on type 'typeof Foo'.

Error: Compiled with errors
```
It generates invalid augmented modules.

**Comment by timocov** [comment]
Yeah looks like a bug. There are 2 test cases for this case:
- https://github.com/timocov/dts-bundle-generator/tree/master/tests/e2e/test-cases/extend-other-module
- https://github.com/timocov/dts-bundle-generator/tree/master/tests/e2e/test-cases/extend-other-module-complex

And it seems that in [this function](https://github.com/timocov/dts-bundle-generator/blob/3170f19e7b8e47475f2de958fac16384ba0f17f8/src/bundle-generator.ts#L537-L564) instead of using `resolveModuleFileName` we need to use something similar to [this](https://github.com/timocov/dts-bundle-generator/blob/3170f19e7b8e47475f2de958fac16384ba0f17f8/src/bundle-generator.ts#L232-L252), but different (not sure if there are existent options for that though).

**Comment by timocov** [comment]
@DerGernTod do you want to work on this issue? just checking to avoid misunderstanding and prevent a situation when several folks are working on the same issue.

**Comment by DerGernTod** [comment]
i'd be glad if anyone else could take this :)

**Comment by timocov** [comment]
Sure. fair enough.

**Comment by timocov** [comment]
The fix has been released in [v6.12.0](https://github.com/timocov/dts-bundle-generator/releases/tag/v6.12.0)
